src/SDKs/DataFactory/DataFactory.Tests/DataFactory.Tests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>DataFactory.Tests</PackageId>
+    <Description>DataFactory.Tests Class Library</Description>
+    <AssemblyName>DataFactory.Tests</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Management.DataFactory\Microsoft.Azure.Management.DataFactory.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="TestData\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.3.0" />
```

Comment:
You need to re record tests using the latest version of Resource Manager at least 1.6.0

Reply:
Brian, who has been working on the C#, is on vacation and neither Hermine nor I understand this comment in his PR that we are trying to complete. Can you help us learn exactly what needs to be done to address it? And is it a blocker for this merge or can it wait?

Reply:
Opened https://github.com/Azure/azure-sdk-for-net/issues/3726  and assigned to BrGold